src/makeCalculator.js

```diff
@@ -4,7 +4,30 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const abilities = {
+    result: 0,
+    add: (number) => (abilities.result += number),
+    subtract: (number) => (abilities.result -= number),
+    multiply: (number) => (abilities.result *= number),
+    divide: (number) => (abilities.result /= number),
+
+    operate: operate,
+    reset: reset,
```

Comment:
[CODE KNOWLEDGE] - When creating methods within an object, you can use the method shorthand syntax. This will make the code cleaner and more concise.

BAD EXAMPLE:
 operate: operate,
 reset: reset,

GOOD EXAMPLE:
 operate,
 reset,

Reply:
take ai comment into consideration